alerter/src/main/java/com/tapadoo/alerter/Alerter.kt

```diff
@@ -753,15 +780,18 @@ class Alerter private constructor() {
          * @return This Alerter
          */
         @JvmStatic
-        fun create(activity: Activity?, @LayoutRes layoutId: Int): Alerter {
+        fun create(activity: Activity?, @LayoutRes layoutId: Int, dialog: AppCompatDialog? = null): Alerter {
```

Comment:
I think this makes for a confusing API. When we have a dialog, there is no need for the Activity. Ultimately, all we require is a reference to the DecorView, in order to add, and remove the Alert. Perhaps storing a weak ref to the only the DecorView is the optimal solution here